mediorum/crudr/setup_test_db.go

```diff
@@ -1,18 +1,22 @@
 package crudr
 
 import (
-	"gorm.io/driver/postgres"
-	"gorm.io/gorm"
+    "gorm.io/driver/postgres"
+    "gorm.io/gorm"
+    "os"
 )
 
 func SetupTestDB() *gorm.DB {
-	dsn := "postgres://postgres:example@localhost:5454/mediorum_test"
-	db, err := gorm.Open(postgres.Open(dsn))
-	if err != nil {
-		panic(err)
-	}
+	dsn := os.Getenv("dbUrl")
```

Comment:
this formatting seems off?

Reply:
Ugh. The tabs vs spaces thing is really screwing with me since we appear to mix and match everywhere